### ISSUE TITLE
Installing theme

### ISSUE THREAD

**Issue by CodingIslander** [alkinkasap/Clean-Blog-Kirby-Theme#1]
Is there a set of instructions that I can follow to install the theme?

**Comment by alkinkasap** [comment]
no just download and extract files, then copy files to your desired location or server. 
there is more about kirby insallation here: https://getkirby.com/docs/installation

after you copied files add /panel end of address to open panel and create user.
then you are ready to go.

**Comment by CodingIslander** [comment]
Based on this, it'd be hard to update the theme sometime in the future - since you include the base code for Kirby.  Or worse, hard to change themes.  Themes and base code should be separate IMO.

**Comment by alkinkasap** [comment]
Actually what you just said is right. But as Kirby is not based on switching themes easily so i didn't feel like i should be taking files apart. 

Still if you're interested you can take theme files out of Kirby. It should be in Kirby Docs.

**Comment by CodingIslander** [comment]
Looks like you currently have Kirby 2.2.0 in your theme.  So I merged the Kirby 2.3.2 /kirby and /panel folders into in your theme to update it to Kirby 2.3.2, then compared it to the base Kirby 2.3.2 install.  Given this info, it didn't look too hard to separate your theme from Kirby's base code - a good start.

I'm doing this because I want a clean install of your theme into a clean Kirby install.

**Comment by alkinkasap** [comment]
yes it's pretty much like that. but i would also suggest you to merge content folder too. as it includes output of default blueprints, unless you already have your content ready.

**Comment by evmguy** [comment]
Blog posts are out of order.
